packages/opentelemetry-cloud-trace-propagator/test/CloudPropagator.test.ts

```diff
@@ -0,0 +1,195 @@
+// Copyright 2020 Google LLC
```

Comment:
The tests are failing due to typing problems.

Reply:
build is fixed now.